Plugin: XDU校园服务

**插件名称：**

XDU校园服务

**插件功能：**

Nonebot2插件，提供基础的西电校园服务，如课表提醒，体育打卡查询及提醒，晨午晚检打卡和马原测试等功能。

**PyPI 项目名：**

nonebot_plugin_xdu_support

**插件 import 包名：**

nonebot_plugin_xdu_support

**插件项目仓库/主页链接：**

https://github.com/longchengguxiao/nonebot_plugin_xdu_support

**标签：**

[{"label":"大学校园","color":"#52b5ea"}]

# 📃 商店发布检查结果

> Plugin: XDU校园服务

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot_plugin_xdu_support 的信息如下：
    name         : nonebot-plugin-xdu-support                                 
     version      : 0.1.6                                                      
     description  : A plugin based on nonebot2, which is support XDU services. 
    
    dependencies
     - libxduauth &gt;=1.7.4
     - lxml *
     - nonebot-adapter-onebot &gt;=2.0.0b1
     - nonebot-plugin-apscheduler &gt;=0.2.0
     - nonebot2 &gt;=2.0.0a16
     - pyDes *
     - pytz *
     - requests *
插件 nonebot_plugin_xdu_support 加载出错：
01-28 05:03:30 [SUCCESS] nonebot | NoneBot is initializing...
01-28 05:03:30 [INFO] nonebot | Current Env: prod
01-28 05:03:30 [SUCCESS] nonebot | Succeeded to import &quot;nonebot_plugin_apscheduler&quot;
01-28 05:03:30 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_apscheduler&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_xdu_support&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_xdu_support/__init__.py&quot;, line 83, in &lt;module&gt;
    require(&quot;nonebot_plugin_apscheduler&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 165, in require
    plugin = load_plugin(name)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 155, in load_plugin
    raise RuntimeError(
RuntimeError: Module nonebot_plugin_apscheduler is not loaded as a plugin! Make sure not to import it before loading.
01-28 05:03:30 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_xdu_support&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_xdu_support&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_xdu_support/__init__.py&quot;, line 83, in &lt;module&gt;
    require(&quot;nonebot_plugin_apscheduler&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 167, in require
    raise RuntimeError(f&#x27;Cannot load plugin &quot;{name}&quot;!&#x27;)
RuntimeError: Cannot load plugin &quot;nonebot_plugin_apscheduler&quot;!</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: 大学校园-#52b5ea。</li><li>✅ 项目 <a href="https://github.com/longchengguxiao/nonebot_plugin_xdu_support">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot_plugin_xdu_support/">nonebot_plugin_xdu_support</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -3971,5 +3971,20 @@
     "homepage": "https://github.com/Proviasw/nonebot-plugin-mcport",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_xdu_support",
+    "project_link": "nonebot_plugin_xdu_support",
+    "name": "XDU校园服务",
+    "desc": "Nonebot2插件，提供基础的西电校园服务，如课表提醒，体育打卡查询及提醒，晨午晚检打卡和马原测试等功能。",
+    "author": "longchengguxiao",
+    "homepage": "https://github.com/longchengguxiao/nonebot_plugin_xdu_support",
+    "tags": [
+      {
+        "label": "大学校园",
+        "color": "#52b5ea"
+      }
+    ],
+    "is_official": false
   }
 ]

